docs/02-app/02-api-reference/04-functions/after.mdx

```diff
@@ -0,0 +1,70 @@
+---
+title: after
+description: API Reference for the after function.
+version: RC
+---
+
+`after()` allows you to schedule work to be executed after a response is finished. This is useful for tasks and other side effects that should not block the response, such as logging and analytics.
+
+It can be used in [Server Components](/docs/app/building-your-application/rendering/server-components) (including [`generateMetadata`](https://nextjs.org/docs/app/api-reference/functions/generate-metadata)), [Server Actions](/docs/app/building-your-application/data-fetching/server-actions-and-mutations), [Route Handlers](/docs/app/building-your-application/routing/route-handlers), and [Middleware](/docs/app/building-your-application/routing/middleware).
+
+The function accepts a callback that will be executed after the response is finished:
+
+```tsx filename="app/layout.tsx switcher
+import { after } from 'next/server'
+import { log } from '@/app/utils'
```

Comment:
Should we add a code comment on this line to indicate that this must come from user code? A quick copy paste will obvs not work for the reader